src/semeio/forward_models/__init__.py

```diff
@@ -40,21 +41,32 @@ def documentation() -> Optional[ForwardModelStepDocumentation]:
             description=design2params_description,
         )
 
+def validate(validation_func):
+    def wrapper(self, fm_step_json):
+        print(f"{fm_step_json=}")
+        should_fail_on_validation_error = (
+            fm_step_json.get("argList")[2].lower() == "true"
```

Comment:
Either the index of the validate arg is sent as a decorator parameter or can we just loop over all parameters if one equals "validate"?